apps/autoscale-by-metric.html.markerb

```diff
@@ -0,0 +1,209 @@
+FAS_APP_NAME="my-app-*"
+```
+
+Then use `$APP_NAME` or `${APP_NAME}` in your Prometheus query to identify the current application being scaled:
+
+```toml
+[env]
+FAS_PROMETHEUS_QUERY = "sum(queue_depth{app='$APP_NAME'})"
+```
+
+You can find a working example of multi-application scaling in the [fly-autoscaler-multiapp-example](https://github.com/fly-apps/fly-autoscaler-multiapp-example) repository.
+
+### Scale based on pending Temporal work
+
+The Temporal metrics collector periodically checks for the total number of workflows in a “running” state. By default, it will check every 15 seconds.
+
+You can connect to your Temporal namespace using the `FAS_TEMPORAL_` environment variables. For example:
+
+```toml
+[env]
+FAS_TEMPORAL_ADDRESS = 'mynamespace.lyeth.tmprl.cloud:7233'
+FAS_TEMPORAL_NAMESPACE = 'mynamespace.lyeth'
+FAS_TEMPORAL_METRIC_NAME = 'queue_depth'
+
+FAS_APP_NAME = "my-target-app"
+FAS_CREATED_MACHINE_COUNT="workflow_count / 10"
+```
+
+In the example above the autoscaler is set up to create or destroy Machines for an app that can handle up to 10 workflows at a time (based on the current workflow count) with: `FAS_CREATED_MACHINE_COUNT="workflow_count / 10`. If you want to ensure you don’t exceed a specific number of Machines, then you can use a `min()` expression to cap it: `FAS_CREATED_MACHINE_COUNT="min(50, workflow_count / 10)`. This ensures that no more than 50 Machines get created, regardless of how many workflows are executing.
+
+You'll also need to load the certificate and key data as secrets from your `ca.pem` and `ca.key` files:
+
+```cmd
+fly secrets set --stage FAS_TEMPORAL_CERT_DATA="$(<ca.pem)"
+```
+
+```cmd
+fly secrets set --stage FAS_TEMPORAL_KEY_DATA="$(<ca.key)"
+```
+
+You can find a full working example of Temporal autoscaling in our [fly-autoscaler-temporal-example](https://github.com/fly-apps/fly-autoscaler-temporal-example) repository.
+
+## Configuration reference
+
+The quickstart describes how to configure the autoscaler with environment variables. You can also configure the autoscaler with a YAML config file if you don't want to use environment variables or if you want to configure more than one metric collector.
+
+See the reference [fly-autoscaler.yml](https://github.com/superfly/fly-autoscaler/blob/main/etc/fly-autoscaler.yml) file for an example and more details.
+
+### Autoscaler config
+
+- `FAS_APP_NAME`: The name of the target app to scale.
+- `FAS_CREATED_MACHINE_COUNT`: An [Expr](https://expr-lang.org/) expression to calculate the number of Machines to create or destroy.
+- `FAS_STARTED_MACHINE_COUNT`: An [Expr](https://expr-lang.org/) expression to calculate the number of Machines to stop or start.
+
+### Prometheus collector
+
+- `FAS_PROMETHEUS_ADDRESS`: The URL of the Prometheus endpoint to query.
+- `FAS_PROMETHEUS_METRIC_NAME`: The local variable name that the metric result will be stored as.
+- `FAS_PROMETHEUS_QUERY`: The Prometheus query to run.
+
+
+### Temporal collector
```

Comment:
I see a `FAS_TEMPORAL_QUERY` variable, but it's not in the Fresh Produce or this doc? What is it and should we include it?